docs/Integrating Your HMI/Developing the HMI UI/index.md

```diff
@@ -145,7 +145,7 @@ Each application is able to maintain a list of menu commands through SDL. This i
 If the user chooses to open the menu, the HMI must send a `UI.OnSystemContext` notification with the `SystemContext` enum: `MENU`. After the user exits the menu, another `UI.OnSystemContext` notification must be sent with the `SystemContext` value: `MAIN`.
 !!!
 
-The contents of the app's menu are populated by the RPC `UI.AddCommand`. Each `UI.AddCommand` received corresponds to an individual menu item. When the user selects a menu item via the UI, the HMI should send a `UI.OnCommand` notification. It is best practice to exit the menu after a user makes a selection from the list of commands.
+The contents of the app's menu are populated by the RPC `UI.AddCommand`. Each `UI.AddCommand` received corresponds to an individual menu item. Menu items may have the same menuName, and as such it is the developer's responsibility to make commands clear to the user and not confusing. When the user selects a menu item via the UI, the HMI should send a `UI.OnCommand` notification. It is best practice to exit the menu after a user makes a selection from the list of commands.
 
```

Comment:
```suggestion
The contents of the app's menu are populated by the RPC `UI.AddCommand`. Each `UI.AddCommand` received corresponds to an individual menu item. When the user selects a menu item via the UI, the HMI should send a `UI.OnCommand` notification. It is best practice to exit the menu after a user makes a selection from the list of commands.

!!! NOTE
Several menu items can have the same menuName. As such, it is the developer's responsibility to make commands clear to the user and not confusing in the case that two commands have the same name. 
!!!
```
Perhaps this would be better as a note?